.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+*       @annajung @difince @tzstoyanov @enyinna1234 @pramodrj07
```

Comment:
Looks like ericwb is missing :) And what about Soumil, most of this code is written by him?  We should give him a credit for that, but I do not know where is the best place to put his name.

Reply:
I left myself off intentionally.  And yes, this file's only purpose is to automatically add people on this list to the review of a PR.